src/Combobox/index.js

```diff
@@ -150,61 +170,69 @@ const Combobox = ({
             if (hasSelectedItem) {
               setInputValue(selectedItem.props.value);
             }
-          },
+    },
         })}
       />
-      <ul
-        className={cc([
-          "nds-combobox-list",
-          "list--reset bgColor--white rounded--bottom",
-          "border--right border--bottom border--left",
-          { "nds-combobox-list--active": isOpen },
-        ])}
-        {...getMenuProps()}
-      >
-        {isOpen &&
-          displayedItems.map((item, index) => {
-            let result = (
-              <li
-                key={`${item}-${index}`}
-                className={cc([
-                  "nds-combobox-heading",
-                  "alignChild--left--center padding--x--s padding--y--xs",
-                ])}
-              >
-                {item}
-              </li>
-            );
-
-            if (isSelectable(item)) {
-              result = (
+      {renderLayer(
```

Comment:
`renderLayer` is an interface to `createPortal` with some extra layout effects attached to determine positioning of the layer